wolfssl/wolfcrypt/tfm.h

```diff
@@ -70,7 +70,10 @@
       #define TFM_X86_64
    #endif
 #endif
-#if defined(TFM_X86_64)
+#if defined(__aarch64__)
+    #define TFM_AARCH_64
```

Comment:
Put in a similar check as above:
#if !defined(TCM_AARCH_64) && !defined(TFM_NO_AM)